packages/common/src/operations.graphql

```diff
@@ -31,6 +31,34 @@ query invoice($id: String!) {
       deliveredDatetime
       hold
       color
+      otherParty {
+        __typename
+        ... on NameNode {
+          __typename
+          id
+          name
+          code
+          isCustomer
+          isSupplier
+        }
+
+        ... on NodeError {
+          __typename
+          error {
+            description
+            ... on DatabaseError {
+              __typename
+              description
+              fullError
+            }
+            ... on RecordNotFound {
+              __typename
+              description
+            }
+          }
+        }
+      }
```

Comment:
This is for the auto complete in the detail view primarily, so we have the name object with the ID etc